### PR TITLE
Continuous Clearing Tool V5.2.0 - Release

### PR DESCRIPTION
**Pull request by karthika-g**

### CODE REVIEW

File: src/TestUtilities/appSettingsSW360IntegrationTest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "FossologyURL": "",
+  "Fossologyurl": "",
```
Comment (sumanthkb44) [inline src/TestUtilities/appSettingsSW360IntegrationTest.json:2]: why this got changed ?

File: src/ArtifactoryUploader/PackageUploadHelper.cs [hunk 2]
```diff
@@ -203,26 +242,39 @@ private static string GetComponentType(Component item)
...
+            }
             else
             {
                 // Do nothing
             }
             return string.Empty;
         }
 
-        private async static Task<AqlResult> GetSrcRepoDetailsForPyPiPackages(Component item, CommonAppSettings appSettings)
+        private async static Task<AqlResult> GetSrcRepoDetailsForPyPiOrConanPackages(Component item, CommonAppSettings appSettings)
```
[the remark this answers is not in the document]
Comment (NihalBarick) [reply src/ArtifactoryUploader/PackageUploadHelper.cs:256]: This is tested and here the conditions are different for Python and Conan

**Comment by sumanthkb44** [comment]
Checked & Approved

**Review by sumanthkb44 — APPROVED** [review]
Approve

**Comment by sumanthkb44** [comment]
Creating New PR for 6.0.0

**Pull request closed without merging**